Sources/ProjectDescription/Cache.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+// TODO: Add documentation
+// TODO: Add unit tests
+public struct Cache: Codable, Equatable {
+    public struct Flavor: Codable, Equatable {
+        public let name: String
+        public let configuration: String
+
+        public static func flavor(name: String, configuration: String) -> Flavor {
+            Flavor(name: name, configuration: configuration)
+        }
+    }
+
+    public let flavors: [Flavor]
+
+    public static func cache(flavors: [Flavor]) -> Cache {
```

Comment:
I'd mention in the documentation that the first in the list will be taken as the default if no flavor is provided.